examples/redis/cookbooks/custom-redis/README.md

```diff
@@ -0,0 +1,128 @@
+# Redis
+
+This recipe installs version Redis 2.8 or later using the Redis installer from redis.io.
+
+
+## Installation
+
+For simplicity, we recommend that you create the cookbooks directory at the root of your application. If you prefer to keep the infrastructure code separate from application code, you can create a new repository.
+
+Our main recipes have the `redis` recipe but it is not included by default. To use the `redis` recipe, you should copy this recipe `custom-redis`. You should not copy the actual `redis ` recipe. That is managed by Engine Yard.
+
+1. Edit `cookbooks/ey-custom/recipes/after-main.rb` and add
+
+      ```
+      include_recipe 'custom-redis'
+      ```
+
+2. Edit `cookbooks/ey-custom/metadata.rb` and add
+
+      ```
+      depends 'custom-redis'
+      ```
+
+3. Copy `examples/redis/cookbooks/custom-redis ` to `cookbooks/`
+
+      ```
+      cd ~ # Change this to your preferred directory. Anywhere but inside the application
+
+      git clone https://github.com/engineyard/ey-cookbooks-stable-v5
+      cd ey-cookbooks-stable-v5
+      cp examples/redis/cookbooks/custom-redis /path/to/app/cookbooks/
+      ```
+
+4. Download the ey-core gem on your local machine and upload the recipes
+
+  ```
+  gem install ey-core
+  ey-core recipes upload --environment <nameofenvironment> --path <pathtocookbooksfolder> --apply
+  ```
+
+## Customizations
+
+All customizations go to `cookbooks/custom-redis/attributes/default.rb`.
+
+### Choose the instance that runs the recipe
+
+By default, the redis recipe runs on a utility instance named "redis". You can change this by modifying `attributes/default.rb`.
+
+#### A. Run Redis on a utility instance with a custom name
+
+* Ensure that these lines are not commented out:
+
+```
+  redis['utility_name'] = 'redis'
```

Comment:
To be consistent with how `utility_name` is used in the other recipes, e.g. thinking_sphinx:
https://github.com/engineyard/ey-cookbooks-stable-v5/blob/ac50d45d8281bb7284a49eda452bf3cf73cba878/cookbooks/thinking_sphinx_3/attributes/default.rb#L24

The recipe is easier to use if you only have to set utility_name, instead of modifying the is_redis_instance logic.